src/components/StokesAnalysis/StokesAnalysisComponent.tsx

```diff
@@ -406,6 +407,52 @@ export class StokesAnalysisComponent extends React.Component<WidgetProps> {
         return scatterColors;
     }
 
+    private closestChannel(channel: number, data: Array<{ x: number, y: number, z?: number }>): number {
+        var mid;
+        var lo = 0;
+        var hi = data.length - 1;
+        while (hi - lo > 1) {
+            mid = Math.floor ((lo + hi) / 2);
+            if (data[mid].z < channel) {
+                lo = mid;
+            } else {
+                hi = mid;
+            }
+        }
+        if (channel - data[lo].z <= data[hi].z - channel) {
+            return data[lo].z;
+        }
+        return data[hi].z;
+    }
+
+    private setPointRadius(data: Array<{ x: number, y: number, z?: number }>, channel: { channelCurrent: number, channelHovered: number }, zIndex: boolean): { x: number, y: number, z?: number } {
```

Comment:
`setPointRadius` doesn't seem like an accurate description, since it's _returning_ something rather than setting it.

Reply:
Possibly with a thicker line and the usual color?